src/server/saved_objects/client/lib/errors.js

```diff
@@ -82,6 +82,19 @@ export function isEsUnavailableError(error) {
 }
 
 
+// 503 - Unable to automatically create index because of action.auto_create_index setting
+const CODE_ES_AUTO_CREATE_INDEX_ERROR = 'SavedObjectsClient/autoCreateIndex';
+export function createEsAutoCreateIndexError() {
```

Comment:
__nit:__ it seems every error here has a corresponding unit test suit, so it would be great if you can add one for this new error type as well.